colcon_devtools/verb/version_check.py

```diff
@@ -31,16 +31,17 @@ def main(self, *, context):  # noqa: D102
         for entry_point in colcon_extension_points.values():
             distributions.add(entry_point.dist)
 
-        pypi = ServerProxy('https://pypi.python.org/pypi')
+        base_url = 'https://pypi.python.org/pypi/{project}/json'
         for dist in sorted(distributions, key=lambda d: d.project_name):
-            versions = pypi.package_releases(dist.project_name)
-            if not versions:
+            req = requests.get(base_url.format(project=dist.project_name))
```

Comment:
Nitpick: please change the variable name to `response`.

Reply:
0aeb78c